imports/wasi_snapshot_preview1/fs_test.go

```diff
@@ -1197,7 +1266,32 @@ func Test_fdReaddir(t *testing.T) {
 				}
 			},
 			buf: 0, bufLen: 27, // length is long enough for one more.
-			cookie:          3,  // previous d_next
+			cookie:          2,  // previous d_next
```

Comment:
I have changed this to 2 because if you send in cookie 2, you expect to get entry 3.